pkg/cmd/cli/describe/describer.go

```diff
@@ -230,6 +234,7 @@ func describeBuildSpec(p buildapi.BuildSpec, out *tabwriter.Writer) {
 		formatString(out, "Build Secrets", strings.Join(result, ","))
 	}
 	if len(p.Source.Images) == 1 && len(p.Source.Images[0].Paths) == 1 {
+		noneType = false
```

Comment:
not sure exactly what you did, but make sure it retains the behavior such that for ==1 it uses one print format, and for >1 it uses the other.